treeherder/log_parser/failureline.py

```diff
@@ -132,7 +132,11 @@ def create_group_result(job_log, line):
         )
     else:
         group, _ = Group.objects.get_or_create(name=group_path[:255])
-        duration = int(line.get('duration', 0))
+        duration = line.get('duration', 0)
+        if type(duration) not in ['float', 'int']:
+            duration = 0
```

Comment:
Should we assume it's 0 when it is not numerical? Or should we use something like `null` to signify that the duration is not known?

Reply:
There is other fallback code directly below which sets it to `0`, and the new code follows this concept.